### PR TITLE
Improve error display in top level handler.

### PR DESCRIPTION
**Pull request by ezyang**
I noticed that in some cases we weren't adding a trailing
newline to error messages arising from "error"; this
patch adds a few newlines where there were none to solve
the problem.

Signed-off-by: Edward Z. Yang <ezyang@cs.stanford.edu>

### CODE REVIEW

File: Cabal/Distribution/Simple/Utils.hs [hunk 1]
```diff
@@ -369,6 +369,8 @@ annotateIO verbosity = modifyIOError f
           . withMetadata NeverMark VerboseTrace verbosity
           $ ioeGetErrorString ioe
 
+
+{-# NOINLINE topHandlerWith #-}
```
Comment (23Skidoo) [inline Cabal/Distribution/Simple/Utils.hs]: Why is this needed?

Comment (ezyang) [inline Cabal/Distribution/Simple/Utils.hs]: It makes GHC recompile less when I change Distribution.Simple.Utils :)

Comment (23Skidoo) [inline Cabal/Distribution/Simple/Utils.hs]: Maybe add a comment to that effect?

**Comment by 23Skidoo** [comment]
Merged, thanks!
